src/main/core/support/configuration.rs

```diff
@@ -16,15 +16,18 @@ use super::units::{self, Unit};
 use crate::cshadow as c;
 use log_bindings as c_log;
 
+const START_HELP_TEXT: &str = "\
+    Run real applications over simulated networks.\n\n\
```

Comment:
Why 2 newlines instead of 1?

Reply:
I just thought it looked better as a separate paragraph.